.github/workflows/build-test.yml

```diff
@@ -1,6 +1,10 @@
 name: build-test
 
-on: [push, pull_request]
+on:
+  pull_request:
+  push:
+    branches:
+      - 'main'
```

Comment:
you don't want the tests to run on feature branches?

Reply:
On a pull request, I was seeing things double up - once for the pull_request trigger and once for the push trigger. Or did you mean something else?

Reply:
But we do that everywhere, as in, once we run things on the PR, and then once the PR is merged.

Reply:
That's because you shouldn't be creating branches on the main repo 😁 if we put it here as is, then the CI wouldn't run on PRs merged into release branches (after merge)